.versionrc.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
`standard-version` doesn't support `.cjs` yet, so I updated to use the `.json` version instead